### PR TITLE
Fix BloomPack unpacking to handle Windows vs Linux (BL-1213)

### PR DESCRIPTION
**Pull request by StephenMcConnel**
BloomPack files created on Windows are storing \ as the directory
separator.  This needs to be converted to / on Linux. ZipFile does
this, but FastZip does not.

### CODE REVIEW

**Comment by sillsdevgerrit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.lsdev.sil.org//job/GitHub-Bloom-Wrapper-master-debug/427/
Test PASSed.

File: src/BloomExe/Collection/BloomPack/BloomPackInstallDialog.cs [hunk 1]
```diff
@@ -50,6 +50,7 @@ private void BeginInstall()
 			using (var zip = new ZipFile(_path))
 			{
 				_folderName = GetRootFolderName(zip);
+				zip.Close();
```
Comment (JohnThomson) [inline src/BloomExe/Collection/BloomPack/BloomPackInstallDialog.cs]: It appears the using could end here?
I'm not sure the Close() is needed at all...isn't the point of the using to Dispose of the zip, which will typically close it?

File: src/BloomExe/Collection/BloomPack/BloomPackInstallDialog.cs [hunk 2]
```diff
@@ -130,16 +130,36 @@ private string GetRootFolderName(ZipFile zip)
...
+			// ZipFile instead of FastZip fixes https://jira.sil.org/browse/BL-1213.
+			using (var zip = new ZipFile(_path))
+			{
+				byte[] buffer = new byte[4096];     // 4K is optimum
+				foreach (ZipEntry entry in zip)
+				{
+					var fullOutputPath = Path.Combine(ProjectContext.GetInstalledCollectionsDirectory(), entry.Name);
+					if (entry.IsDirectory)
+						Directory.CreateDirectory(fullOutputPath);
+					if (!entry.IsFile)
```
Comment (JohnThomson) [inline src/BloomExe/Collection/BloomPack/BloomPackInstallDialog.cs]: Is there something else it could be, if it's not a directory?

**Comment by JohnThomson** [comment]
LGTM...couple of minor comments, may not be worth acting on.

**Comment by sillsdevgerrit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.lsdev.sil.org//job/GitHub-Bloom-Wrapper-master-debug/432/
Test PASSed.

**Pull request merged**